.github/workflows/wheels.yml

```diff
@@ -120,7 +115,7 @@ jobs:
         if: ${{ env.IS_32_BIT == 'true' }}
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.3.1
+        uses: pypa/cibuildwheel@v2.4.0
```

Comment:
There are reports that v2.4.0 bumped the setuptools version, causing build problems: https://github.com/pypa/cibuildwheel/issues/1078.

@lithomas1 I was reminded today that if we build with vs2019, we need to do some work to make sure it links with the 14.1 library instead of the default 14.2 library. Do you know if that works here?

Reply:
I think I'll need to vendor the script from MacPython/numpy-wheels. I'll try that in another PR. 

Regarding setuptools version, I think that is pinned in pyproject.toml. At the moment, the wheel builds pass right now, so I guess no action needed there for now?

Reply:
Makes sense.